codex-ui/dev/Playground.vue

```diff
@@ -7,15 +7,15 @@
         { label: 'Light', value: 'light' },
         { label: 'Dark', value: 'dark' },
       ]"
-      value="dark"
+      :value="colorMode"
       name="color-scheme"
-      @change="($event) => setColorScheme($event.target.value)"
+      @change="($event) => setColorMode($event.target.value)"
```

Comment:
The CSS Working Group has standardized the use of `color-scheme` for specifying preferred color schemes.